pkg/storaged/clevis-luks-passphrase.sh

```diff
@@ -1,7 +1,13 @@
-#! /bin/bash
+#! /bin/sh
 
 set -eu
 
+type=no
```

Comment:
Please add a comment what this means. As the usual noun ("data type") a yes/no makes no sense. As a verb ("hacking on the keyboard") it does, but it'd be nice to document the behaviour.

Reply:
Would `opt_type` be enough?

Reply:
To me it was quite obvious that it is an option, but not what it actually *means*. So I'd prefer a comment (of course you can rename it to opt_ also)

Reply:
Done.

Reply:
Ah, I see.  I have added a full help text.